WeScanTests/VisionRectangleDetectorTests.swift

```diff
@@ -36,28 +37,61 @@ final class VisionRectangleDetectorTests: FBSnapshotTestCase {
 
     containerLayer.render(in: UIGraphicsGetCurrentContext()!)
 
-    let image = UIGraphicsGetImageFromCurrentImageContext()!
+    self.image = UIGraphicsGetImageFromCurrentImageContext()!
```

Comment:
It's general good practice to reset in the tearDown as well. Might be good here as well to include:

```swift
image = nil
containerLayer = nil
```

It's already set up correctly for that! 👌